migrate/20250425_victoria_metrics_resource.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_table(:victoria_metrics_resource) do
+      column :id, :uuid, primary_key: true
+      column :name, :text, null: false
+      column :created_at, :timestamptz, null: false, default: Sequel.lit("now()")
+      column :admin_user, :text, collate: '"C"', null: false
+      column :admin_password, :text, collate: '"C"', null: false
+      column :target_vm_size, :text, collate: '"C"', null: false
+      column :target_storage_size_gib, :bigint, null: false
+      column :root_cert_1, :text, collate: '"C"'
+      column :root_cert_key_1, :text, collate: '"C"'
+      column :root_cert_2, :text, collate: '"C"'
+      column :root_cert_key_2, :text, collate: '"C"'
+      column :certificate_last_checked_at, :timestamptz, null: false, default: Sequel.lit("now()")
+      foreign_key :project_id, :project, type: :uuid, null: false
+      foreign_key :location_id, :location, type: :uuid, null: false
+    end
+    create_table(:victoria_metrics_server) do
+      column :id, :uuid, primary_key: true
+      column :name, :text, null: false
```

Comment:
Do we really need a name for the servers?

Reply:
It's just for informational purpose for now, if we move to cluster version we'll have a bit more information here to make it easier to identify different server types